### PR TITLE
Update user guide

### LINKED ISSUE #184: Docs-UG: Update introductory section
We should update the introductory section to follow the rubrics provided.

image

### LINKED ISSUE #187: Docs-UG: Remove CLI commands from Quick Start
Non-technically savvy users might be intimidated upon seeing commands in the quick start guide. 

Let's move these instructions to the FAQ.

image

### PR DESCRIPTION
**Pull request by dillontkh**
Resolves #184, resolves #187

### CODE REVIEW

**Review by RyanNgWH — APPROVED** [review]
Changes looks good!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -235,9 +288,25 @@ _Details coming soon ..._
 
 ## FAQ
 
-**Q**: How do I transfer my data to another Computer?<br>
+###### How do I transfer my data to another Computer?
+
 **A**: Install FitBook in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous FitBook home folder.
 
+###### What version of Java do I have?
```
Comment (bgopi23) [inline docs/UserGuide.md:295]: thanks for adding more FAQ questions

File: docs/UserGuide.md [hunk 2]
```diff
@@ -248,12 +317,12 @@ _Details coming soon ..._
...
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [nt/NOTE] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+| Action     | Format, Examples                                                                                                                                                                                    |
+|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**    | `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [nt/NOTE] [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 nt/likes pizzas t/friend t/colleague` |
+| **Clear**  | `clear`                                                                                                                                                                                             |
```
Comment (bgopi23) [inline docs/UserGuide.md:323]: very nice formatting

File: docs/UserGuide.md [hunk 3]
```diff
@@ -2,40 +2,90 @@
...
-        1. Open a command prompt (**cmd** for Windows, **Terminal** for Mac)
+[//]: # (Continue numbering from 2, as the block above interrupts the list and resets the number)
 
-        1. Use `cd` to navigate to the folder containing FitBook.jar
-            * e.g. `cd ~/Downloads/FitBook/`
+{:start="2"}
+1. Download the latest version of `FitBook.jar` from <a href="https://github.com/AY2324S2-CS2103T-T17-3/tp/releases" target="_blank">here</a>.
+
+1. Copy the file to the folder you want to use as the **home folder** for FitBook.
+   * FitBook will save all the data it needs this folder.
```
Comment (bgopi23) [inline docs/UserGuide.md:78]:
```suggestion
   * FitBook will save all the data it needs in this folder.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -211,8 +263,9 @@ There is no need to save manually.
 FitBook data is saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. 
 Advanced users are welcome to update data directly by editing that data file.
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, FitBook will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.<br>
+<div markdown="block" class="alert alert-warning">:warning: **Warning:**
+If your changes to the data file makes its format invalid, FitBook will discard all data and start with an empty data file at the next run. Hence, it is recommended to make a backup of the file before editing it.
```
Comment (bgopi23) [inline docs/UserGuide.md:267]:
```suggestion
If your changes to the data file make its format invalid, FitBook will discard all data and start with an empty data file at the next run. Hence, it is recommended to make a backup of the file before editing it.
```

**Review by bgopi23 — CHANGES_REQUESTED** [review]
LGTM other then a few nits

**Review by bgopi23 — APPROVED** [review]
LGTM!

**Pull request merged**